src/components/FeeCollapse/index.tsx

```diff
@@ -21,35 +20,29 @@ interface CollapseProps {
 }
 
 export const FeeCollapse: FC<CollapseProps> = ({ toAmount, toToken, exchangeRate }) => {
-  const [isOpen, setIsOpen] = useState(false);
   const { trackEvent } = useEventsContext();
+  const isOpenCheckboxRef = useRef<HTMLInputElement>(null);
   const toTokenSymbol = toToken.fiat.symbol;
 
-  const toggleIsOpen = () => {
+  const trackFeeCollapseOpen = () => {
     trackEvent({ event: 'click_details' });
-    setIsOpen((state) => !state);
   };
 
-  const chevron = isOpen ? (
-    <ChevronUpIcon className="w-8 text-blue-700" />
-  ) : (
-    <ChevronDownIcon className="w-8 text-blue-700" />
-  );
-
-  const toAmountFixed = roundDownToSignificantDecimals(new Big(toAmount || 0), 2).toString();
+  const toAmountFixed = roundDownToTwoDecimals(Big(toAmount || 0)).toString();
```

Comment:
`roundDownToTwoDecimals` already returns a string (and this occurs two more times below).